include/console.h

```diff
@@ -13,16 +13,17 @@
 extern "C" {
 #endif
 
-/** @brief Initialize console_getchar() call.
+/** @brief Initialize console_getchar()/putchar() calls.
  *
  *  This function should be called once to initialize pull-style
- *  access to console via console_getchar() function. This function
- *  supersedes, and incompatible with, callback (push-style) console
- *  handling (via console_input_fn callback, etc.).
+ *  access to console via console_getchar() function and buffered
+ *  output using console_putchar() function. This function supercedes,
+ *  and incompatible with, callback (push-style) console handling
+ *  (via console_input_fn callback, etc.).
  *
  *  @return N/A
  */
-void console_getchar_init(void);
```

Comment:
Is there a concern about us removing an API, not sure if we can support a deprecated version or not?